metricbeat/module/linux/pressure/_meta/fields.yml

```diff
@@ -0,0 +1,90 @@
+- name: pressure
+  type: group
+  release: beta
+  description: >
+    Linux pressure stall information metrics for cpu, memory, and io
+  fields:
+    - name: pressure
+      type: group
+      description: Linux pressure stall information
+      fields:
+        - name: cpu.some.10
```

Comment:
In `system/process` cgroup data, we're reporting these as `10.pct` and so on. We'll also need to add `format: percent`

Reply:
Cool. I tried to follow similar conventions for `total`, in that it becomes `total.time.us` - does this make sense here?